src/routes/forms.js

```diff
@@ -253,6 +285,7 @@ export default [
  * @typedef {import('@defra/forms-model').FormMetadataAuthor} FormMetadataAuthor
  * @typedef {import('~/src/api/types.js').RequestFormById} RequestFormById
  * @typedef {import('~/src/api/types.js').RequestRemoveFormById} RequestRemoveFormById
+ * @typedef {import('~/src/api/types.js').RequestFormMetadataUpdateById} RequestFormMetadataUpdateById
```

Comment:
Just a reminder we can start using the new `@import` syntax now TypeScript 5.5 is installed

```js
/**
 * @import { FormMetadataAuthor, FormMetadataInput } from '@defra/forms-model'
 * @import { AuthCredentials, RequestAuth, ServerRoute, UserCredentials } from '@hapi/hapi'
 * @import { OidcStandardClaims } from 'oidc-client-ts'
 * @import { RequestFormById, RequestFormBySlug, RequestFormDefinition, RequestFormMetadataCreate, RequestFormMetadataUpdateById, RequestRemoveFormById } from '~/src/api/types.js'
 */
```

Reply:
I will keep that in mind. I think I am going to leave it in this PR and change all in separate PR, if that's okay?

Reply:
Yeah course, thanks Kiran

Reply:
as agreed, will be addressed in different PR. I added to my todo list